src/test/libcephfs/snapdiff.cc

```diff
@@ -56,7 +59,58 @@ class TestMount {
     return ceph_conf_get(cmount, option, buf, len);
   }
 
-  string make_file_path(const char* relpath) {
+  int rank0_config_set(const char * var, const char * val) {
+    char *outb,*outs;
+    size_t outb_len, outs_len;
+
+    char cmd[1024];
+    snprintf(cmd, sizeof(cmd) / sizeof(char), R"({"prefix":"config set","var":"%s","val":["%s"]})", var, val);
+    const char* cmdv[] = {cmd};
+    int status = ceph_mds_command(cmount, "0", cmdv, 1, nullptr, 0, &outb, &outb_len, &outs, &outs_len);
+    if (status < 0) {
+      std::cout << "couldn't set config option '" << var << "': " << strerror(-status) << " outs: " << outs << std::endl;
+    }
+    return status;
+  }
+
+  int rank0_config_unset(const char *var){
+    char *outb, *outs;
+    size_t outb_len, outs_len;
+
+    char cmd[1024];
+    snprintf(cmd, sizeof(cmd) / sizeof(char), R"({"prefix":"config unset","var":"%s"})", var);
```

Comment:
`config rm` is widely used, so let's stick to using that.

Reply:
```
couldn't tell rank 0 '{"format":"json","prefix":"config rm","var":"mds_max_caps_per_client"}'
Invalid argument: unknown command prefix config rm
```
I'll stick to `config unset` for now

Reply:
OK. I see what you mean

```
ceph tell mds.c --help | grep config
...
...
config unset <var>                                                                                                                     config unset <field>: unset a config variable
...
...
```

While

```
ceph config rm mds mds_max_caps_per_client
```

is just fine.

Reply:
So I can't use this version (i.e. config rm ...) with the interface of `mds_command`, can I?

Reply:
`ceph config` changes configs managed by the mon config store (a global database of configuration and somewhat new)

the asok/tell commands are all routed to the MDS as an "mds_command" which provides a different interface for setting configs on the particular daemon, overriding everything else.